src/utils/log_metrics.py

```diff
@@ -122,3 +122,21 @@ def std_var_events_in_log(log: EventLog) -> int:
     :return 3
     """
     return statistics.stdev([len(trace) for trace in log])
+
+
+def trace_ids(log: EventLog) -> list:
+    values = []
+    for trace in log:
+        values.append(trace.attributes[NAME_CLASSIFIER])
+    return values
+
+
+def traces(log: EventLog) -> list:
```

Comment:
once again should be:
def traces(log: EventLog) -> list:
return [ {'attributes': trace.attributes, 'events': [event for event in trace]} for trace in log]